expertise/service/expertise.py

```diff
@@ -357,21 +168,21 @@ def get_expertise_all_status(self, user_id):
             self.logger.info(f"Attempting to load {search_dir}/config.json")
             with open(os.path.join(search_dir, 'config.json'), 'r') as f:
                 s = f"{''.join(f.readlines())}"
-                config = json.loads(s)
-            status = config['status']
-            description = config['description']
+                config = JobConfig.from_json(json.loads(s))
```

Comment:
```suggestion
                config = JobConfig.from_json(json.load(f))
```

No need to convert to string first.